dev/src/dev/render_png.clj

```diff
@@ -65,16 +65,15 @@
     (.deleteOnExit tmp-file)
     (open tmp-file)))
 
-(defn open-hiccup-as-html [hiccup]
-  (let [html-str (hiccup/html hiccup)
+(defn open-hiccup-as-html [hiccup]  (let [html-str (hiccup/html hiccup)
```

Comment:
Looks like there is no change in this file except for formatting. Not sure if this is intended, but want to let you know in case, this is unintentional.